### PR TITLE
Link to FAQ and remove "experimental"

### PR DESCRIPTION
**Pull request by Poopooracoocoo**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ In addition, most mods depend on an API mod, containing commonly used hooks. The
 
 * [Fabric API for Minecraft 1.14 and above](https://minecraft.curseforge.com/projects/fabric/files)
 
+It is recommended that you read the <a href="https://fabricmc.net/wiki/faq:user">Frequently Asked Questions for users.</a> Modding-friendly alternative launchers such as MultiMC are recommended over Minecraft Launcher.
```
Comment (modmuss50) [inline index.md:20]: See my comment: https://github.com/FabricMC/fabricmc.net/pull/41#pullrequestreview-1381059206

File: index.md [hunk 2]
```diff
@@ -13,9 +13,14 @@ Fabric is a lightweight, experimental modding toolchain for Minecraft.
 
 To install Fabric, you need the **Fabric Loader**, which you can <a class="page-link page-link-download" href="/use">download here</a>
 
-In addition, most mods depend on an API mod, containing commonly used hooks. They can be found here:
+You should also read the [Fabric Wiki installation instructions](https://fabricmc.net/wiki/install)!
```
Comment (modmuss50) [inline index.md:16]: Doesnt read quite right to me (Although I could be wrong), maybe:

```suggestion
You can also read the [installation instructions on the Fabric Wiki](https://fabricmc.net/wiki/install)!
```

File: index.md [hunk 3]
```diff
@@ -13,9 +13,14 @@ Fabric is a lightweight, experimental modding toolchain for Minecraft.
...
 
-* [Fabric API for Minecraft 1.14 and above](https://minecraft.curseforge.com/projects/fabric/files)
+In addition, most mods depend on the Fabric API mod, containing commonly used hooks. It must be added to your mods folder. Download Fabric API for Minecraft 1.14 and above from here:
+
+* [CurseForge download for Fabric API](https://minecraft.curseforge.com/projects/fabric/files)
+* [Modrinth download for Fabric API](https://modrinth.com/mod/fabric-api/versions)
```
Comment (modmuss50) [inline index.md:21]: Id prefer to keep just the CurseForge download link here.

File: index.md [hunk 4]
```diff
@@ -13,9 +13,14 @@ Fabric is a lightweight, experimental modding toolchain for Minecraft.
...
+
+It is recommended that you read the <a href="https://fabricmc.net/wiki/faq:user">Frequently Asked Questions for users.</a>
```
Comment (modmuss50) [inline index.md:23]: Why use an `<a>` tag here, and mark down above?

File: index.md (hunk 2 above)
Comment (Poopooracoocoo) [reply index.md]: Personally I feel like users _should_ read it, as a recommendation, but I respect that you've made a decision! :)

File: index.md (hunk 4 above)
Comment (Poopooracoocoo) [reply index.md:23]: Oops! Would be great if you could make these changes. I'm going to have to go now xD

Comment (modmuss50) [reply index.md:23]: No worries, I think I will want to checkout these changes first and double check they look good on a built site. Its quite hard to review/visualise HTML 👍

File: index.md (hunk 3 above)
Comment (Poopooracoocoo) [reply index.md:21]: Why do you prefer to only have a CurseForge link? Is it the primary download? 

Still looks pretty clean and I believe that having a link to Modrinth will reduce support requests. Modrinth's featured versions are separate for every major version, such as 1.19.x, which would reduce support requests about having the wrong version. A lot of users come to Fabric for mods like Sodium or Iris, which are now only available on Modrinth, and would reduce support requests.

Comment (modmuss50) [reply index.md:21]: A few reasons:

- I have some major concerns about Modrith's security, only the other day I asked if they had a Bug Bounty program and found out they didnt. I decided to have a quick poke around and in 10 mins (no joke) I had found a critical security flaw that effecively allowed me to remove any (even ones I dont own) from the site. I reported this and it was fixed quickly, but I believe the whole site needs a lot of work to improve things in this regard.
- The new CurseForge website makes it a lot easier to find the correct version.
- (Id be lying if I didnt mention this) Fabric isnt free to run, the revenue I made from CurseForge currently covers these costs, Modrith doesnt even come anywhere close to covering the running costs.

By linking to it we basically give it a seal of approval, right now in its current state I am not happy enough with it to do this. I currently have no plans to remove Fabric API from there.

Comment (Poopooracoocoo) [reply index.md:21]: I see. I understand and respect that. Feel free to make those changes!! :)

**Pull request closed without merging**